.circleci/config.yml

```diff
@@ -330,6 +330,22 @@ jobs:
                 name: Tag
                 command: |
                   ./ops/scripts/ci-docker-tag-op-stack-release.sh <<parameters.registry>>/<<parameters.repo>> $CIRCLE_TAG $CIRCLE_SHA1
+      - when:
+          condition: "<<parameters.release>>"
+          steps:
+            - gcp-oidc-authenticate:
+                service_account_email: GCP_SERVICE_ATTESTOR_ACCOUNT_EMAIL
+            - run:
+                name: Sign
+                command: |
+                  SIGNER_FOLDER="./.circleci/signer"
```

Comment:
Any reason this is hardcoded in the command and not passed in via env var config thru circleci directly?

Reply:
This is a constant value, and in line with the rest of the configuration that use hardcoded file path.